pages/page.py

```diff
@@ -55,55 +62,36 @@ def select_option(self, value, locator):
             raise Exception("Option '" + value + "' was not found, thus not selectable.")
 
     def is_element_present(self, *locator):
-        self.selenium.implicitly_wait(0)
         try:
             self.selenium.find_element(*locator)
             return True
         except NoSuchElementException:
-            # this will return a snapshot, which takes time.
             return False
-        finally:
-            # set back to where you once belonged
-            self.selenium.implicitly_wait(self.testsetup.default_implicit_wait)
 
     def is_element_visible(self, *locator):
```

Comment:
I notice you enhanced `wait_for_element_visible` to allow for either an element or a locator to be passed. Why not do the same for `is_element_visible`?